koekalenteri-shared/model/Event.ts

```diff
@@ -0,0 +1,27 @@
+export type Event = {
+    id: string
+    "event-type": string
+    "classes": [string]
+    "start-date": Date
+    "end-date": Date
+    "location": string
+    "name": string
+    "description": string
+    "allow-online-entry": boolean
+    "allow-online-payment": boolean
+    "unofficial": boolean
+    "allow-owner-membership-priority": boolean
+    "allow-handler-membership-priority": boolean
+    "cost": number
+    "cost-member": number
+    "payment-details": string
+    "account-number": string
+    "reference-number": string
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/koekalenteri/koekalenteri/pullRequest?prid=7376473)
```suggestion
    'reference-number': string
```